docs/toc.yml

```diff
@@ -607,6 +607,9 @@
     - name: Shapes
       href: excel/excel-add-ins-shapes.md
       displayName: Excel
+    - name: Show or hide an Office Add-in in a shared runtime
+      href: excel/show-hide-add-in.md
+      displayName: Excel
```

Comment:
Why is this in the Excel section (same goes for the **Run code on document open** article)? It seems like even if they currently only work on Excel, they belong in the **Core Concepts** realm.

Reply:
I'm following the lead of @davidchesnut in the placement in the TOC for all the shared runtime-related docs. I'll leave it where it is for now and confer with him.